tsdef/index.d.ts

```diff
@@ -0,0 +1,36 @@
+import { ConnectOptions, CreateLocalTrackOptions, CreateLocalTracksOptions, LocalTrack } from './types';
+import { LocalAudioTrack } from './LocalAudioTrack';
+import { LocalVideoTrack } from './LocalVideoTrack';
+import { Room } from './Room';
+
+export const isSupported: boolean;
+export const version:string;
+export function connect(token: string, options?: ConnectOptions): Promise<Room>;
+export function createLocalAudioTrack(options?: CreateLocalTrackOptions): Promise<LocalAudioTrack>;
+export function createLocalTracks(options?: CreateLocalTracksOptions): Promise<LocalTrack[]>;
+export function createLocalVideoTrack(options?: CreateLocalTrackOptions): Promise<LocalVideoTrack>;
+
+export { AudioTrack } from './AudioTrack';
+export { LocalTrack, RemoteTrack, LogLevel, LogLevels } from './types';
+export { LocalAudioTrack } from './LocalAudioTrack';
+export { LocalAudioTrackPublication } from './LocalAudioTrackPublication';
+export { LocalDataTrack } from './LocalDataTrack';
+export { LocalDataTrackPublication } from './LocalDataTrackPublication';
+export { LocalParticipant } from './LocalParticipant';
+export { LocalTrackPublication } from './LocalTrackPublication';
+export { LocalVideoTrack } from './LocalVideoTrack';
+export { LocalVideoTrackPublication } from './LocalVideoTrackPublication';
+export { Participant } from './Participant';
```

Comment:
`Participant` is a template class, so you can't export that. Instead you can export a type:
```suggestion
type Participant = LocalParticipant | RemoteParticipant;
export Participant;
```

Reply:
I can put this in types and export it out of there. I think it will look a little cleaner!